cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,48 @@
+/// <reference types='cypress' />
+
 describe('', () => {
-  before(() => {
+  let user;
 
+  beforeEach(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+      cy.login(user.email, user.username, user.password);
+      cy.visit('/');
+    });
   });
 
-  it('', () => {
-
+  it('should have the ability to create an article', () => {
+    let article;
+    let usernameToLowerCase = user.username.toLowerCase();
+    cy.task('generateArticle').then((generateArticle) => {
```

Comment:
It would be better to generate an article in the beforeEach part to avoid writing it in every test

Reply:
Thank you for your advice. When I need to write 10 tests, I'll take it out 😉